### PR TITLE
CAMEL-12209: Proposal to fix by adding an overload

### PR DESCRIPTION
**Pull request by aldettinger**
Please check [CAMEL-12209](https://issues.apache.org/jira/browse/CAMEL-12209) out for more details.

### CODE REVIEW

File: camel-core/src/main/java/org/apache/camel/util/StringHelper.java [hunk 1]
```diff
@@ -711,17 +711,28 @@ public static boolean containsIgnoreCase(String src, String what) {
...
         }
         int exp = (int) (Math.log(bytes) / Math.log(unit));
         String pre = "KMGTPE".charAt(exp - 1) + "";
-        return String.format("%.1f %sB", bytes / Math.pow(unit, exp), pre);
+        return String.format(locale, "%.1f %sB", bytes / Math.pow(unit, exp), pre);
+    }
+
+    /**
+     * Outputs the bytes in human readable format in units of KB,MB,GB etc.
+     *
```
Comment (aldettinger) [inline camel-core/src/main/java/org/apache/camel/util/StringHelper.java]: I could add a comment there to explicit that it relies on the default locale.

Comment (davsclaus) [reply camel-core/src/main/java/org/apache/camel/util/StringHelper.java]: Good idea, also maybe add a `@see` to that other method where you can specify the exact locale to use.

**Comment by oscerd** [comment]
Closing since it has been merged

**Comment by aldettinger** [comment]
Oops, I've totally missed this one. @oscerd, thanks for closing 👍

**Comment by oscerd** [comment]
np :+1: glad to help!

**Pull request closed without merging**